Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -373,6 +373,8 @@ def pr_check_conflicts(user, repo, pr)
       "org:#{user}", repo: repo, state: "open", label: "\"no long build conflict\""
     ).each_with_object({}) do |long_build_pr, hash|
       number = long_build_pr["number"]
+      next if number == pr
```

Comment:
```suggestion
      next if number == pr.to_i
```

Types in this file are an absolute mess.

Reply:
Thanks!